packages/@azure/servicebus/data-plane/test/namespace.spec.ts

```diff
@@ -323,3 +326,78 @@ describe("Errors with non existing Queue/Topic/Subscription", async function():
     should.equal(errorWasThrown, true, "Error thrown flag must be true");
   });
 });
+
+describe("Test createFromAadTokenCredentials", function(): void {
+  let namespace: Namespace;
+  let tokenCreds: msrestAzure.ApplicationTokenCredentials;
+  let errorWasThrown: boolean = false;
+  if (!process.env.SERVICEBUS_CONNECTION_STRING) {
+    throw new Error(
+      "Define SERVICEBUS_CONNECTION_STRING in your environment before running integration tests."
+    );
+  }
+  const serviceBusEndpoint = (process.env.SERVICEBUS_CONNECTION_STRING.match(
+    "Endpoint=sb://(.*)/;.*"
```

Comment:
We don't need the `.*` in the end.

Reply:
Also the `/` before the `;` is optional. 
I would suggest to use the regular expression as `Endpoint=sb://((.*).servicebus.windows.net)`